.circleci/config.yml

```diff
@@ -37,6 +39,13 @@ templates:
           - v2-artifacts-{{ .Revision }}
 
 jobs:
+  build:
```

Comment:
is this job needed?

Reply:
yes for now, circleci demands to at least have 1 job called build, is just made it to circleci happy (this can go when we ditch circleci)